### DISCUSSION TITLE
Some extra features (proposal)

### DISCUSSION THREAD

**Discussion by GeoSot** [lubusIN/laravel-decomposer#37] [Ideas]
Hi . You have done a great Job

If you would like to add some more interesting features on your project, i guess some caching on the values and a config file would be great

Some of the values cost too much time and resources so they could be cached for sometime or by session (maybe a choice config file)

And in a config file you could add the route and some middleware that anyone wish to add, plus the pagination number

Finally you can give the choice to full integrate the decomposer view, in every project. The view could be  published in order to by able to modify.

And an adition of the last minute
On some servers, for security reasons some function are disabled. (ex:php_uname)
As a result the code brakes an gets errors
It could be checked with  ini_get('disable_functions')  before use

**Comment by ajitbohra** [comment]
@GeoSot, sorry for the delayed response, & thanks for some good suggestions makes sense. Most of them overlap with what we are planning for our future release plan which is currently in the planning stage https://github.com/orgs/lubusIN/projects/18
